server/src/main/scala/org/scassandra/server/priming/json/ResultJsonRepresentation.scala

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
I moved this into it's own json subpackage as some of the class names were clashing, but wasn't sure if this was a good idea, what do you think?